<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConstraintMeshWidget</class>
 <widget class="QWidget" name="CConstraintMeshWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Force stage modulation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="stageCheckBox_0">
        <property name="text">
         <string>Stage 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="stageCheckBox_1">
        <property name="text">
         <string>Stage 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="stageCheckBox_2">
        <property name="text">
         <string>Stage 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="stageCheckBox_3">
        <property name="text">
         <string>Stage 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="vertexColorLightingCheckBox">
     <property name="text">
      <string>Force vertex color lighting</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Texture anim type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="texAnimTypeComboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Global</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="reinitCheckBox">
     <property name="text">
      <string>Reinit when new element is created</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="stageLabel">
     <property name="text">
      <string>Stage:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="stageSpinBox">
     <property name="maximum">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Translation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="transUStartDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>V Start:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="transVStartDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>U Speed:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="transUSpeedDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>V Speed:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="transVSpeedDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>U Accel:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="transUAccelDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>V Accel:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="transVAccelDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>U Start:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Scale</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>U Start:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="scaleUStartDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>V Start:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="scaleVStartDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>U Speed:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="scaleUSpeedDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>V Speed:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="scaleVSpeedDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>U Accel:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="scaleUAccelDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>V Accel:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="scaleVAccelDoubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Rotation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="rotSpeedDoubleSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Accel:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="rotAccelDoubleSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>144</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
